rust/BUILD.md

```diff
@@ -142,7 +142,11 @@ Also see [compare-features.sh](./scripts/compare-features.sh) which when provide
 
 ### Perf + Firefox Profiler
 
-TODO
+```sh
+cargo install --locked samply
+# same steps as for Perf + Flamegraph but instead of `perf script | ...` run 
```

Comment:
This is unfortunately not enough, I do not know what should I use instead of ...
It is not clear how to open the Firefox profiler, neither. (and in the next paragraph as well)